src/components/StakingVaults/DeFiEarn.tsx

```diff
@@ -40,11 +42,14 @@ export const DeFiEarn: React.FC<DefiEarnProps> = ({
   header,
   ...rest
 }) => {
+  const { isConnected } = useWallet().state
   const { q } = useQuery<{ q?: string }>()
   const [searchQuery, setSearchQuery] = useState(q ?? '')
   const translate = useTranslate()
   const [selectedChainId, setSelectedChainId] = useState<ChainId | undefined>()
-  const portfolioChainIds = useAppSelector(selectWalletConnectedChainIdsSorted)
+  const chainIds = useAppSelector(state =>
+    isConnected ? selectWalletConnectedChainIdsSorted(state) : knownChainIds,
+  )
```

Comment:
Allows DeFi chain dropdown to show for all chains since wallet doesn't narrow chains... there is no wallet here